### ISSUE TITLE
Gaps in K8up metrics

### ISSUE THREAD

**Issue by simu** [k8up-io/k8up#976]
### Description

Currently, K8up only emits timeseries for schedules for which it's seen at least one Job with the matching completion state, e.g. `k8up_jobs_successful_counter` will only have timeseries for schedules which have at least one successful job since the last K8up restart.
 
For schedules with relatively low frequency (e.g. 1/day) this can lead to significant gaps in the metric in Prometheus which confuses Prometheus functions such as `rate()` which otherwise can compensate for counter resets due to pod restarts.

### Expected Behavior

K8up initializes the counter metrics (`k8up_jobs_failed_counter`, `k8up_jobs_successful_counter`, and `k8up_jobs_total`) with value 0 for all job types and all namespaces in which a Schedule exists immediately after startup.

### Version of K8up

v2.7.2

### Version of Kubernetes

v1.27.13

### Distribution of Kubernetes

OpenShift 4

**Comment by mhutter** [comment]
For my understanding, the problem would be fixed if K8up would emit those metrics with labels for all existing schedules and a value of 0?

**Comment by simu** [comment]
> For my understanding, the problem would be fixed if K8up would emit those metrics with labels for all existing schedules and a value of 0?

Yes, (if I understand Prometheus's behavior correctly) emitting metrics with labels for all existing schedules and value 0 until the first job is observed would let Prometheus correctly identify the counter resets.

**Comment by arska** [comment]
The counter metrics in `operator/monitoring/prometheus.go` are `CounterVec` with labels `{namespace, jobType}`. Prometheus only emits a timeseries once a label combination has been used (via `.WithLabelValues().Inc()`), which is why counters don't appear until the first job runs.

A fix would add an `InitCounters` function that:
1. Lists all Schedule resources across namespaces
2. For each schedule's namespace, calls `.WithLabelValues(namespace, jobType).Add(0)` for each job type

This would need to be called at operator startup after the informer cache syncs. The function itself is straightforward — the complexity is wiring it into the right place in the startup sequence.
